notebooks/3-Model-pipeline.ipynb

```diff
@@ -49,7 +49,7 @@
     }
```

Comment:
<p>The confusion matrix seems to me quite similar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/esciencecenter-digital-skills/SICSS-odissei-machine-learning/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>